docs/wiki/Verification_and_Validation_Plan.md

```diff
@@ -0,0 +1,133 @@
+# 1 Introduction
+This verification and validation plan applies to the NASA Operational Simulator (NOS).  NOS is described in more detail in the User's Manual and Developer's Guide and section 3.
+
+## 1.1 Purpose and Scope
+NOS is a suite of tools developed by NASA's Katherine Johnson Independent Verification and Validation (IV&V) Facility to aid in areas such as software development, integration & test (I&T), mission operations/training, verification and validation (V&V), and software systems check-out. NOS provides a software development environment, a multi-target build system, an operator interface/ground station, dynamics and environment simulations, and software-based models of spacecraft hardware.
+
+The purpose of this plan is to identify the activities that will establish compliance with the requirements (verification) and to establish that the system will meet the customers' expectations (validation).
+
+## 1.2 Responsibility and Change Authority
+The NASA Independent Verification and Validation (IV&V) Independent Test Capability (ITC) team is responsible for maintenance of this plan.  This group has the authority to approve changes to this plan.
+
+# 2 Applicable and Reference Documents
+## 2.1 Applicable Documents
+1. Concept of Operations / Architecture
+2. User's Manual and Developer's Guide
+3. Comprehensive Performance Tests
+4. Limited Performance Tests
+5. Requirements
+6. Requirements Traceability Matrix (RTM)
```

Comment:
Can we make stubs of these places if they don't already exist? Not sure if it's possible to link to them as well. Maybe @rjbrown2 would now how to do that.